src/components/draft-js-editor/DraftJSEditor.js

```diff
@@ -117,13 +120,17 @@ class DraftJSEditor extends React.Component<Props> {
                     {label}
                     {!isRequired && <OptionalFormattedMessage />}
                 </span>
+                <span className={classNames({ 'accessibility-hidden': true })} id={this.commentFormTip}>
+                    <FormattedMessage {...messages.atMentionTipDescription} />
+                </span>
 
                 <Tooltip isShown={!!error} position="bottom-left" text={error ? error.message : ''} theme="error">
                     {/* need div so tooltip can set aria-describedby */}
                     <div>
                         <Editor
                             {...a11yProps}
                             ariaLabelledBy={this.labelID}
+                            ariaDescribedBy={this.commentFormTip}
```

Comment:
DraftJsEditor is a generic editor - whereas this `commentFormTip` id naming & the atMentionTipDescription message text seems specific to [CommentForm](https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/activity-feed/comment-form/CommentForm.js#L136). Can you find a way to generalize this so the ariaDescribeBy message can be owned/passed in by the parent component

Reply:
Sure, nice catch!